src/system_tray.cpp

```diff
@@ -268,5 +276,74 @@ namespace system_tray {
     return 0;
   }
 
+  void
+  update_tray_playing(std::string app_name) {
+    tray.notification_title = NULL;
+    tray.notification_text = NULL;
+    tray.notification_cb = NULL;
+    tray.notification_icon = NULL;
+    tray.icon = TRAY_ICON_PLAYING;
+    tray_update(&tray);
+    tray.icon = TRAY_ICON_PLAYING;
+    tray.notification_title = "Stream Started";
+    char msg[256];
+    sprintf(msg, "Streaming Started for %s", app_name.c_str());
+    tray.notification_text = msg;
+    tray.notification_icon = TRAY_ICON_PLAYING;
```

Comment:
Can you update the tooltip for the playing and paused states, just so it's clear to users what the icon means?